materia-app.Dockerfile

```diff
@@ -65,15 +65,15 @@ RUN composer install --no-cache --no-dev --no-progress --no-scripts --prefer-dis
 # =====================================================================================================
 FROM node:12.11.1-alpine AS yarn_stage
 
-RUN apk add --no-cache git
+RUN apk add --no-cache git python make g++
```

Comment:
I ran into issues on new base php images building that were helped by adding these